### PR TITLE
Add activate endpoint and activate field in db with test cases

### PR DESCRIPTION
**Pull request by ChungLinWu0829**
### Description:
Add activate endpoint and activate field in db with test cases

### Test result and coverage:
image
image

### Lint check
image

### Story reference: 
https://app.zenhub.com/workspaces/promotions-su24-001-666234565dda150276ad19c5/issues/gh/csci-ga-2820-su24-001/promotions/9

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ class Promotion(db.Model):  # pylint: disable=too-many-instance-attributes
     modified_by = db.Column(db.Uuid, nullable=True)
     created_when = db.Column(db.DateTime, nullable=False)
     modified_when = db.Column(db.DateTime, nullable=True)
+    activate = db.Column(db.Boolean, nullable=False, default=False)
```
Comment (mingyi850) [inline service/models.py:80]: Can we change to 'active' instead of 'activate' - just so it's a noun

**Review by mingyi850 — CHANGES_REQUESTED** [review]
Looks good! Just one comment

File: service/models.py (hunk 1 above)
Comment (ChungLinWu0829) [reply service/models.py]: Sorry for the oversight, done!

Comment (mingyi850) [reply service/models.py]: No worries, thanks man!

**Review by mingyi850 — APPROVED** [review]
LGTM

**Pull request merged**